tests/sample/bindmonitor.c

```diff
@@ -67,24 +67,27 @@ find_or_create_process_entry(bind_md_t* ctx)
     int index;
 
     entry = bpf_map_lookup_elem(&process_map, &key);
-    if (entry)
+    if (entry) {
         return entry;
+    }
 
-    if (ctx->operation != BIND_OPERATION_BIND)
+    if (ctx->operation != BIND_OPERATION_BIND) {
         return entry;
+    }
 
-    if (!ctx->app_id_start || !ctx->app_id_end)
+    if (!ctx->app_id_start || !ctx->app_id_end) {
         return entry;
+    }
 
     bpf_map_update_elem(&process_map, &key, &value, 0);
     entry = bpf_map_lookup_elem(&process_map, &key);
-    if (!entry)
+    if (!entry) {
         return entry;
+    }
 
 #if 0
     for (index = 0; index < 64; index++) {
-        if ((ctx->app_id_start + index) >= ctx->app_id_end)
-            break;
+        if ((ctx->app_id_start + index) >= ctx->app_id_end) {             break; }
```

Comment:
move break to new line

Reply:
Strange that clang formatting didn't catch this.

Reply:
OK, this is commented out code, which doesn't get formatted automatically.